ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -643,6 +643,17 @@ tests:
         TEST=./src/tests/test_upgrade_agent.py
         TEST_FUNC=test_upgrade_agent
     workflow: assisted-baremetal
+- as: e2e-nutanix-assisted
+  cluster: build01
+  optional: true
+  steps:
+    cluster_profile: nutanix
+    env:
+      ASSISTED_CONFIG: |
+        NUM_WORKERS=0
+        MAKEFILE_TARGET=test
+      PLATFORM: nutanix
+    workflow: assisted-nutanix
```

Comment:
Add this test also to `assisted-service` configurations.
Also add a daily periodic job (start it around 1:00 AM)

Reply:
This best be added in a separate PR. We only allow 3 clusters due to quotas in Nutanix CI